services/121-service/src/scripts/seed-helper.ts

```diff
@@ -291,8 +291,13 @@ export class SeedHelper {
 
   public async assignAdminUserToProgram(programId: number): Promise<void> {
     const userRepository = this.dataSource.getRepository(UserEntity);
+    const adminEmail = process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN
+      ? Equal(process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN)
+      : undefined;
     const adminUser = await userRepository.findOneOrFail({
-      where: { username: process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN },
+      where: {
+        username: adminEmail,
+      },
```

Comment:
I think it would be safer to assume this is always defined. Also, the point of this change is to never assign `undefined` inside the `where` clause to avoid the [original bug](https://github.com/typeorm/typeorm/issues/2500#issuecomment-1916774575) (which is why we are making these changes in the first place).

```suggestion
      where: {
        username: Equal(process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN!),
      },
```